build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -876,6 +890,166 @@ message SymlinkNode {
   NodeProperties node_properties = 4;
 }
 
+// A `RelaxedFileTree` represents a node in a file tree, containing zero or
+// more sub
+// [RelaxedFileTreeNodes][build.bazel.remote.execution.v2.RelaxedFileTree] and
+// zero or more children
+// [RelaxedFileNodes][build.bazel.remote.execution.v2.RelaxedFileNode],
+// [RelaxedDirectoryNodes][build.bazel.remote.execution.v2.RelaxedDirectoryNode]
+// and [RelaxedSymlinkNodes][build.bazel.remote.execution.v2.RelaxedSymlinkNode].
+// Each `Node` contains either its absolute or relative path, either the digest
+// of its content (either a file blob or a `RelaxedFileTree` proto) or a
+// symlink target, as well as possibly some metadata about the file or
+// directory.
+//
+// The paths MUST use `/` as path component separator, even if the environment
+// where the Action is executed use other separators. If the path starts with
+// `/` it is considered to be refering to the root of the file tree, otherwise
+// the path is relative to the parent directory. The path MUST NOT contain any
+// `.` or `..` components. An empty string refers to the same directory.
```

Comment:
Given that the `RelaxedFileTree` contains a base `path`, one can require all children paths to be relative to that base path.